dwave_networkx/generators/chimera.py

```diff
@@ -495,3 +495,83 @@ def tuple_pairs(self, plist):
             Equivalent to (tuple(self.tuples(p)) for p in plist)
         """
         return self.__pair_repack(self.tuples, plist)
+
+def linear_to_chimera(r, m, n=None, t=None):
+    """Convert the linear index `r` into a chimera index.
+
+
+
+    Parameters
+    ----------
+    r : int
+        The linear index value.
+    m : int
+        Number of rows in the Chimera lattice.
+    n : int (optional, default m)
+        Number of columns in the Chimera lattice.
+    t : int (optional, default 4)
+        Size of the shore within each Chimera tile.
+
+
+    Returns
+    -------
+    i : int
+        The column of the Chimera index's unit cell associated with `r`.
+    j : int
+        The row of the Chimera index's unit cell associated with `r`.
+    u : int
+        Whether the index is even (0) or odd (1); the side of the bi-partite graph of the Chimera unit cell.
+    k : int
+        Index into the Chimera unit cell.
+
+    Examples
+    --------
+    >>> G = dnx.linear_to_chimera(0, 8, 8, 4)
```

Comment:
Should be a space between the `------` and the `>>> ...`, also the example should show the output. E.g.
```
Examples
--------

>>> G = dnx.linear_to_chimera(0, 8, 8, 4)
(0, 0, 0, 0)
```

Reply:
Do we want to make the spacing congruent with this in the Returns section, or just put the blank line here?

Reply:
I think it only needs to be there for examples, but consistency is good!